packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -1,7 +1,7 @@
 import Log from 'sap/base/Log';
 import type AppLifeCycle from 'sap/ushell/services/AppLifeCycle';
 import type { InitRtaScript, RTAPlugin, StartAdaptation } from 'sap/ui/rta/api/startAdaptation';
-import scenarios, { type Scenario } from 'sap/ui/fl/Scenario';
+import { scenario as scenarios, type Scenario } from '@sap-ux-private/control-property-editor-common';
```

Comment:
That looks odd. Why do we have this singular vs. plurar change?

Reply:
This is used in the method that accepts the scenario as an argument:

https://github.com/SAP/open-ux-tools/blob/8493f7fc3ff72629d8f2c55b0dceaf90bed74329/packages/preview-middleware-client/src/flp/init.ts#L198

What would be a better name here?

Reply:
In the code after that I need to declare a `scenario` variable and there was a clash of the names. As the import here is actually a structure constant, containing all available scenarios, to me it made sense to be plural.

Reply:
It is a constant, so it should actually be `SCENARIO`. Could you please fix it at the source and update all its references.

@voicis or would you disagree?

Reply:
@tobiasqueck I agree with your proposal, but do you think it is better to this refactoring in a different PR?

Reply:
I am ok with a different one but then do the different one first, so that we don't add that odd usage here.

Reply:
I think it would be better to fix it here. 
Fixed with: 8447f1c3c9a83860ddf0726fe5124896444d03f4

Reply:
Looks good to me, now, I would like to see @voicis approving this PR.